packages/fxa-admin-panel/src/components/AccountSearch/ConnectedServices/index.tsx

```diff
@@ -0,0 +1,175 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import dateFormat from 'dateformat';
+import { AttachedClient, Location } from 'fxa-admin-server/src/graphql';
+import React from 'react';
+import { DATE_FORMAT, ResultTableRow } from '../Account';
+
+type Nullable<T> = T | null;
+
+export const NUMBER_OF_SERVICES_TO_SHOW = 3;
+
+export const ConnectedServices = ({
+  services,
+}: {
+  services?: Nullable<AttachedClient[]>;
+}) => {
+  if (services && services.length > 0) {
+    if (services.length > NUMBER_OF_SERVICES_TO_SHOW) {
+      return (
+        <details>
+          <summary className="hover:cursor-pointer text-violet-900 font-semibold mb-4">
+            Toggle viewing {services.length} connected services
```

Comment:
More a matter of a preference than even a nit, but I would probably say something like "Show {services.length} connected services" for clarity. But 🤷 that might just be specific to my brain

Reply:
Heh, I had that at first, but then we'd need a `useState` to account for when it's open to then change the "Show" text to "Hide". Could do that, but wasn't sure it was worth it. 😂